frontend/packages/console-app/console-extensions.json

```diff
@@ -465,6 +465,12 @@
       ]
     }
   },
+  {
+    "type": "console.flag/hookProvider",
+    "properties": {
+      "handler": { "$codeRef": "userPreferences.usePerspectivesAvailable" }
+    }
+  },
```

Comment:
Does this flag get reevaluated when other plugins load or only once?

I think we might have a race if plugins contribute perspectives but are loaded asynchronously.